<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="BOM-5CG1085LY2" failures="1" tests="2" name="Default test(failed)" time="73.533" errors="0" timestamp="2023-11-19T03:09:19 IST">
  <testcase classname="com.amazon.Hawk.TestSuites.BuyTestSuite" name="addToCartAndVerify" time="40.278"/>
  <testcase classname="com.amazon.Hawk.TestSuites.BuyTestSuite" name="cancelProductAndVerify" time="33.18">
    <failure type="java.lang.AssertionError" message="Order is not cancelled expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Order is not cancelled expected [true] but found [false]
at com.amazon.Hawk.TestSuites.BuyTestSuite.cancelProductAndVerify(BuyTestSuite.java:95)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- cancelProductAndVerify -->
  <testcase classname="com.amazon.Hawk.TestSuites.BuyTestSuite" name="buyProductAndVerify" time="0.0">
    <ignored/>
  </testcase> <!-- buyProductAndVerify -->
</testsuite> <!-- Default test(failed) -->
